great_expectations/core/batch_definition.py

```diff
@@ -72,9 +72,6 @@ def get_batch(self, batch_parameters: Optional[BatchParameters] = None) -> Batch
 
         return batch_list[-1]
 
-    def save(self) -> None:
-        self.data_asset._save_batch_definition(self)
-
```

Comment:
Not actually implemented - let's remove it until we can revisit this hierarchy later